DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -426,7 +430,7 @@ FROM Candidates
             {
                 logger.LogWarning(
                     "Not enrolled delegate on self assessment as db insert failed. " +
-                    $"Self assessment id: {selfAssessmentId}, candidate id: {candidateId}"
+                    $"Self assessment id: {selfAssessmentId}, user id: {delegateUserId}"
                 );

```

Comment:
Super-pedantic but personally I'd have put this as 'delegate user id' so future debuggers know we aren't talking about say an admin user id here. :)